inc/client_communication.hpp

```diff
@@ -173,24 +177,52 @@ class PackedClientEntry : public ClientEntryAbstract {
 
 };
 
+int8_t ParseMsg(uint8_t* rx_data, uint8_t rx_length,
+  ClientEntryAbstract** entry_array, uint8_t entry_length);
+
+int8_t ParseMsg(uint8_t* rx_data, uint8_t rx_length,
```

Comment:
Where is this second signature for ParseMsg used? I didn't notice it around anywhere, may be missing it